### PR TITLE
Add `gleam-nibble` to the list of parser projects

### PR DESCRIPTION
**Pull request by jiegillet**
I recently picked up Gleam for the Advent of Code, so I looked up `pd-andy/gleam-string-parser` which redirects to `hayleigh-dot-dev/gleam-string-parser` but I couldn't `gleam add` it.
Looking around on the repo, I found `hayleigh-dot-dev/gleam-nibble` which was exactly what I was looking for, it's a great library that deserves to be on this awesome list.

### CODE REVIEW

**Review by lpil — COMMENTED** [review]
This is the renamed string-parser. Would you mind removing that old entry also?

File: README.md [hunk 1]
```diff
@@ -150,6 +150,7 @@ Libraries for parsing different kinds of data.
 - [delucks/pathlib](https://github.com/delucks/pathlib) - Manipulate Unix-style filesystem paths.
 - [sporto/gleam_codec](https://github.com/sporto/gleam_codec) - Build codecs (encode and decode) to transform data structures between Gleam and Elixir.
 - [pd-andy/gleam-string-parser](https://github.com/pd-andy/gleam-string-parser) - A simple parser combinator package written in Gleam.
```
Comment (jiegillet) [inline README.md:152]:
```suggestion
```

**Comment by jiegillet** [comment]
Done!

**Review by lpil — APPROVED** [review]
Thank you!

**Pull request merged**